src/renderers/webgl-fallback/WebGLBackend.js

```diff
@@ -1082,18 +1120,18 @@ class WebGLBackend extends Backend {
 					data[ 0 ] = i;
 
 					gl.bindBuffer( gl.UNIFORM_BUFFER, bufferGPU );
-					gl.bufferData( gl.UNIFORM_BUFFER, data, gl.DYNAMIC_DRAW );
+					gl.bufferData( gl.UNIFORM_BUFFER, data, gl.STATIC_DRAW );
```

Comment:
The camera index buffers are just updated once and then only bound/unbound. In this case, `STATIC_DRAW` is the better choice.